redball/upgrade.py

```diff
@@ -445,4 +445,15 @@ def upgrade_database(toVer=None):
             time.time()
         ),
     ],
+    14: [
+        # Add Sidebar Updater bot type
```

Comment:
Nice, you saved me the step of doing this... It looks like you forgot to update the comment after copy/pasting though.